executor/executor.spec

```
@@ -0,0 +1,101 @@
+from PyInstaller.utils.hooks import collect_all, collect_submodules
+
+block_cipher = None
+
+# Collect all dependencies
+datas = []
+binaries = []
+hiddenimports = []
+
+# Collect shared module
+shared_path = os.path.join(os.path.dirname(os.getcwd()), 'shared')
+if os.path.exists(shared_path):
```

RuboCop (1.81.7)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)